### PR TITLE
Add values files from last deployment

### PR DESCRIPTION
**Pull request by sbtaylor15**
Changed trigger on cloudbuild side to ignore these updates to avoid infinite build loop.

See ortelius/ortelius#378

### CODE REVIEW

File: chart/ms-general/values.yaml [hunk 1]
```diff
@@ -1,43 +1,36 @@
...
+dockertag: svc_cat-v10.0.0.492-g6d649a4
+dropzone: /tmp/tdm.1292.1
+environment: aks-cluster
+gitcommit: 6d649a4
+gitrepo: ortelius/ortelius
+gittag: svc_cat
+giturl: git@github.com:ortelius/ortelius.git
+kubectl_context: cluster2-aks-admin
+server_hostname: localhost
+server_name: localhost
```
Comment (markjacksonfishing) [inline chart/ms-general/values.yaml]: Line feed at the end.

File: chart/ms-nginx/values.yaml [hunk 2]
```diff
@@ -1,48 +1,36 @@
...
+dockertag: svc_cat-v10.0.0.492-g6d649a4
+dropzone: /tmp/tdm.1292.1
+environment: aks-cluster
+gitcommit: 6d649a4
+gitrepo: ortelius/ortelius
+gittag: svc_cat
+giturl: git@github.com:ortelius/ortelius.git
+kubectl_context: cluster2-aks-admin
+server_hostname: localhost
+server_name: localhost
```
Comment (markjacksonfishing) [inline chart/ms-nginx/values.yaml]: Same line feed at the end

File: chart/ms-ui/values.yaml [hunk 3]
```diff
@@ -1,43 +1,36 @@
...
+dockertag: svc_cat-v10.0.0.492-g6d649a4
+dropzone: /tmp/tdm.1292.1
+environment: aks-cluster
+gitcommit: 6d649a4
+gitrepo: ortelius/ortelius
+gittag: svc_cat
+giturl: git@github.com:ortelius/ortelius.git
+kubectl_context: cluster2-aks-admin
+server_hostname: localhost
+server_name: localhost
```
Comment (markjacksonfishing) [inline chart/ms-ui/values.yaml]: Line feed at the end.

**Review by markjacksonfishing — CHANGES_REQUESTED** [review]
Is there a reason the values are not being templatized?

**Comment by sbtaylor15** [comment]
This is data that is needed by ArgoCD Application Sets to perform a deployment into the cluster via GitOps process.  Ortelius/DeployHub will be generating this down the road as part of the pipeline.  Just need a starting point to get ArgoCD setup completed.

**Review by markjacksonfishing — APPROVED** [review]

**Pull request merged**